samples/helloworld/ext2rootfs/Makefile

```diff
@@ -28,7 +28,9 @@ package.pem:
 
 package: package.pem ext2rootfs
 	echo "Generating a signed package"
-	@myst package-sgx --roothash=roothash appdir package.pem ../config.json
+	@myst package-sgx --roothash=roothash package.pem ../config.json
```

Comment:
What does it mean when both roothash and appdir are present in args? Should this be prevented(we should not change what we have, just wondering..)

Reply:
The TEE_aware sample also takes appdir as a command line arg